python/requirements.txt

```diff
@@ -17,6 +17,7 @@ pyyaml
 aiosignal
 frozenlist
 requests
+watchfiles
```

Comment:
should we pin this?

Reply:
no need to pin it, we are using a public/stable interface